packages/server/src/server.ts

```diff
@@ -118,14 +127,16 @@ const logAndHandleResponse = async (methodName, methodFunction) => {
   const messagePrefix = `[POST] ${methodName}:`;
   try {
     const response = await methodFunction();
-    const success = response instanceof JsonRpcError ? 'Failure' : 'Success';
+    const success = response instanceof JsonRpcError ? responseFailureLabel : responseSuccessLabel;
```

Comment:
`success=Success` and `success=Failure` are repetitive and don't make sense or provide details. Instead suggest `statusCode=-32600` or `status=Invalid request|Success|...`.

Reply:
I had been thinking on this when pushing the PR out, was planning on addressing in a separate ticket.
Will capture the status code though

Reply:
Updated